themes/custom/apigee_kickstart/src/components/card/api-card.twig

```diff
@@ -0,0 +1,43 @@
+{#
+/**
+ * @file
+ * Template for a Card component.
+ */
+#}
+{% set classes = [
+  'card',
+  'api-card'
+]|merge(classes|default([])) %}
```

Comment:
We should make the other card component support what needs to happen here and use it in a follow up.  This is ok right now.